Themis/Views/Assessment/CorrectionSidebar/GradingCriteriaCellView.swift

```diff
@@ -5,26 +5,60 @@
 //  Created by Florian Huber on 20.11.22.
 //
 
+// swiftlint:disable line_length
+
 import Foundation
 import SwiftUI
 
 struct GradingCriteriaCellView: View {
-    let gradingCriterium: GradingCriteria
+    let gradingCriterion: GradingCriterion
 
     var body: some View {
+
         VStack(alignment: .leading) {
-            ForEach(gradingCriterium.structuredGradingInstructions) { instruction in
-                HStack {
-                    Text(String(format: "%.1f", instruction.credits))
-                        .font(.title3)
-                    Spacer()
-                    Text(instruction.gradingScale)
-                        .font(.title3)
-                    Spacer()
-                    Text(instruction.instructionDescription)
-                        .font(.title3)
+            if let gradingCriterionTitle = gradingCriterion.title {
+                Text(gradingCriterionTitle).font(.title3)
+            }
+
+            ForEach(gradingCriterion.structuredGradingInstructions) { instruction in
+                VStack {
+                    HStack {
+                        Text(instruction.gradingScale)
+                            .font(.title3)
+                        Spacer()
+                        Text(String(format: "%.1f", instruction.credits))
+                            .font(.title3)
+                    }
+
+                    HStack {
+                        Text(instruction.instructionDescription)
+                            .font(.title3)
+
+                        Spacer()
+                    }
                 }
```

Comment:
The second HStack with Spacer is unnecessary.
```suggestion
                VStack(alignment: .leading) {
                    HStack {
                        Text(instruction.gradingScale)
                            .font(.title3)
                        Spacer()
                        Text(String(format: "%.1f", instruction.credits))
                            .font(.title3)
                    }
                        Text(instruction.instructionDescription)
                            .font(.title3)
                }
```